### ISSUE TITLE
Logging Loss

### ISSUE THREAD

**Issue by Michalos88** [google/dopamine#66]
Hello Dopamine Team,

Happy New Year!

I am trying to log the loss using the Tensorflow summary module for various agents.
I saw that you build a wrapper `summary_writer` to capture most of the data, but in case of the losses you used the native implementation. Why?

For example in line 299 of dqn_agent.py:
`tf.summary.scalar('HuberLoss', tf.reduce_mean(loss))`
and in line 357 of implicit_quantile_agent.py:
`tf.summary.scalar('QuantileLoss', tf.reduce_mean(loss))`

I believe that `tf.summary.merge_all()` should merge all of the summaries, but for some reason I can't get it to log the loss. 

Perhaps it's saved to file separately from Evals, but then where? 

I'd really appreciate your help!
Thank you for supporting this project!

**Comment by Michalos88** [comment]
Hi all, 

I realized that losses will only be logged in the debug mode. 
So if you want to capture the loss in tf.summary, you have to pass a flag '--debug_mode=True'.

I believe a note about debug mode should be added to the documentation, as dopamine supposed to be a research framework for fast prototyping of new reinforcement learning algorithms. 

Thanks
-Michal

**Comment by mgbellemare** [comment]
Hi Michal,

That's a fair point. Mostly we've been using the logger to record experimental results and Tensorboard to debug new ideas, but it makes sense that this aspect of --debug_mode should be renamed more accurately --output_tf_summaries or such.
